src/Symfony/Component/Console/Output/ConsoleOutput.php

```diff
@@ -153,14 +153,18 @@ private function openOutputStream()
             return fopen('php://output', 'w');
         }
 
-        return @fopen('php://stdout', 'w') ?: fopen('php://output', 'w');
+        return \defined('STDOUT') ? \STDOUT : (@fopen('php://stdout', 'w') ?: fopen('php://output', 'w'));
     }
 
     /**
      * @return resource
      */
     private function openErrorStream()
     {
-        return fopen($this->hasStderrSupport() ? 'php://stderr' : 'php://output', 'w');
+        if (!$this->hasStderrSupport()) {
+            return fopen('php://output', 'w');
```

Comment:
Shall we use `STDOUT` here as well if it's defined?

Reply:
php://output !== php://stdout, so not sure

Reply:
True. 🤔